bootique-shiro-web-jwt/src/main/java/io/bootique/shiro/web/jwt/ShiroWebJwtModuleFactory.java

```diff
@@ -80,6 +80,10 @@ public JwtBearerAuthenticationFilter createFilter(Provider<JwtParser> tokenParse
         return new JwtBearerAuthenticationFilter(tokenParser, this.audience);
     }
 
+    public String provideAudience() {
```

Comment:
Ah, I forgot that the new filter will also need `audience`. I suppose we can keep it here for now, but after we merge the PR, I am thinking of refactoring both `shirowebjwt` and `shiroweboidconnect` config structure, so that we don't need to share properties (likely going back to your original idea of nested configurations were properties are organized by "filter", "callback", "jwks", etc.). 

Anyways, no change is needed in this PR